api/src/pcapi/core/educational/models.py

```diff
@@ -38,6 +38,17 @@ class StudentLevels(enum.Enum):
     GENERAL0 = "Lycée - Terminale"
 
 
+student_mapping = {
```

Comment:
Dans le code backend, on a plutôt l'habitude de nommer les constantes en majuscule : 

```suggestion
STUDENT_MAPPING = {
```

(Je me demande si ça ne devrait d'ailleurs pas s'appeler "student _level_ mapping", mais je ne connais pas trop le contexte...)

Reply:
... d'autre part, je comprends que ce sont des labels qui viennent d'Adage, c'est bien ça ? Si oui, peut-être que le nom de la variable devrait l'indiquer, pour éviter par exemple qu'on ré-utilise ce mapping à des endroits qui ne sont pas concernés ? Suggestion : 

```suggestion
ADAGE_STUDENT_LEVEL_MAPPING = {
```

Reply:
Aligné!
Je prends le fix

Reply:
J'arrive après la guerre tant pis héhé, mais ca vient du front de pro et pas d'adage :) du coup peut etre rechanger le naming pour enlever adage ?